test/lib/mayaUsd/render/pxrUsdMayaGL/testProxyShapeDrawUsdChangeProcessing.py

```diff
@@ -84,39 +99,6 @@ def _RunTest(self, dagPathName):
         self.assertEqual(prim.GetTypeName(), 'Cube')
         self._WriteViewportImage(self._testName, 'Cube')
 
-    def testUsdChangeProcessingAssembly(self):
-        """
-        Tests that authoring on a USD stage that is referenced by an assembly
-        node (in "Collapsed" representation with a proxy shape underneath)
-        invokes drawing refreshes in Maya correctly.
-        """
-        self._testName = 'UsdChangeProcessingTest_Assembly'
-
-        mayaSceneFile = '%s.ma' % self._testName
-        mayaSceneFullPath = os.path.abspath(mayaSceneFile)
-        cmds.file(mayaSceneFullPath, open=True, force=True)
-
-        UsdMaya.LoadReferenceAssemblies()
-
-        assemblyDagPathName = '|UsdChangeProcessingTest|Primitive'
-        self._RunTest(assemblyDagPathName)
```

Comment:
The assembly case was removed, so everything was collapsed from `_RunTest()` into `testUsdChangeProcessingProxy()`.